website/content/userguide/permanent-data.markdown

```diff
@@ -0,0 +1,120 @@
+---
+title: "Access permanent data"
+menuTitle: "Access permanent data"
+weight: 49
+---
+
+## Create the filesystem
+
+gokrazy provides a handy way to create the permanent data filesystem on the forth partition of your gokrazy installation.
+The [gokrazy/mkfs](https://github.com/gokrazy/mkfs) program will do all the required steps to create the filesystem.
```

Comment:
will do all the required steps to create the filesystem.

→ will create the filesystem.